internal/renderer/service_util_test.go

```diff
@@ -649,6 +650,251 @@ func TestRenderServiceUtil(t *testing.T) {
 				assert.Equal(t, string(gw.Spec.Listeners[0].Port), string(sp[0].Port), "sp 1 - port")
 			},
 		},
+		{
```

Comment:
Thanks, PRs with tests are always welcome. Can we also add a test where we have annotations both on the GwConf and the Gw and see whether the latter correctly override the former? No need to test all cases, just the basics...